.github/workflows/master.yaml

```diff
@@ -15,9 +15,9 @@ jobs:
     - name: Checkout
       uses: actions/checkout@v1
     - name: Lint
-      uses: actions-contrib/golangci-lint@master
+      uses: golangci/golangci-lint-action@v2
       with:
-        args: run
+        version: v1.39.0
```

Comment:
version should not be specified, and args should be `run -p bugs`

Reply:
Please take a close look to my fixes for the warnings caused by initial addition of "-p bugs"